### PR TITLE
blog(embertimes128): accessible ember sortable

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-12-13-the-ember-times-issue-128.md [hunk 1]
```diff
@@ -24,6 +25,14 @@ READMORE
 
 ---
 
+## [Accessible controls for ember-sortable 🎛️](https://www.linkedin.com/pulse/open-source-a11y-deep-dive-ember-sortable-yicheng-jerry-gong/)
```
Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]:
```suggestion
## [Accessible Controls for ember-sortable 🎛️](https://www.linkedin.com/pulse/open-source-a11y-deep-dive-ember-sortable-yicheng-jerry-gong/)
```

File: source/2019-12-13-the-ember-times-issue-128.md [hunk 2]
```diff
@@ -24,6 +25,14 @@ READMORE
...
+
+[@ygongdev](https://github.com/ygongdev) wrote a [blog post](https://www.linkedin.com/pulse/open-source-a11y-deep-dive-ember-sortable-yicheng-jerry-gong/) that documented the process on how we now have [accessible controls](https://github.com/adopted-ember-addons/ember-sortable/issues/269) for [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) an open source drag and drop list sort Ember addon.
```
Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]:
```suggestion
[Yicheng (Jerry) Gong (@ygongdev)](https://github.com/ygongdev) wrote a [blog post](https://www.linkedin.com/pulse/open-source-a11y-deep-dive-ember-sortable-yicheng-jerry-gong/) that documented the process on how we now have [accessible controls](https://github.com/adopted-ember-addons/ember-sortable/issues/269) for [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable), an open source drag and drop list sort Ember addon.
```

File: source/2019-12-13-the-ember-times-issue-128.md [hunk 3]
```diff
@@ -24,6 +25,14 @@ READMORE
...
+
+The functionality needed to be added was thanks to the comprehensive list on what accessible features should be expected on a drag and drop utility and even a [CodePen demo](https://codepen.io/drewlee/project/full/XWNLeE) from [@drewlee](https://github.com/drewlee). The accessible controls have landed as part of the [2.0.0 release](https://github.com/adopted-ember-addons/ember-sortable/releases/tag/v2.0.0) of the [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) addon. Check out the accessible controls at the [demo page](https://adopted-ember-addons.github.io/ember-sortable/demo/) today!
```
Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]:
```suggestion
The functionality needed to be added was thanks to the comprehensive list on what accessible features should be expected on a drag and drop utility and even a [CodePen demo](https://codepen.io/drewlee/project/full/XWNLeE) from [Andrew A Lee (@drewlee)](https://github.com/drewlee). The accessible controls have landed as part of the [2.0.0 release](https://github.com/adopted-ember-addons/ember-sortable/releases/tag/v2.0.0) of the [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) addon. Check out the accessible controls at the [demo page](https://adopted-ember-addons.github.io/ember-sortable/demo/) today!
```

Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]: not sure what you mean in this first sentence so didn't do "suggestion" ... perhaps this?

`The functionality was added thanks to the comprehensive list on what accessible features should be expected on a drag...`

File: source/2019-12-13-the-ember-times-issue-128.md [hunk 4]
```diff
@@ -24,6 +25,14 @@ READMORE
...
+
+[ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) is also now part of the [adopted-ember-addons](https://github.com/adopted-ember-addons) which is a GitHub org where community members can find a [new home](https://github.com/adopted-ember-addons/program-guidelines/blob/master/README.md) for their Ember addon.
```
Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]:
```suggestion
[ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) is also now part of the [adopted-ember-addons](https://github.com/adopted-ember-addons), which is a GitHub org where community members can find a [new home](https://github.com/adopted-ember-addons/program-guidelines/blob/master/README.md) for their Ember addon.
```

Comment (amyrlam) [inline source/2019-12-13-the-ember-times-issue-128.md]: could add something mentioning the december blog post: `ember-sortable was also highlighted on [DecEmber Day 1](https://blog.emberjs.com/2019/12/01/countdown-to-the-new-year-ember-sortable.html)! `

**Review by amyrlam — APPROVED** [review]
looks great, thank you! 👍

left some optional suggestions, hopefully not too confusing 

there was a CI error, so ended up rebasing the parent blog branch

File: source/2019-12-13-the-ember-times-issue-128.md (hunk 3 above)
Comment (chrisrng) [reply source/2019-12-13-the-ember-times-issue-128.md]: Thanks updated the wording :) dunno what i was saying

File: source/2019-12-13-the-ember-times-issue-128.md (hunk 4 above)
Comment (chrisrng) [reply source/2019-12-13-the-ember-times-issue-128.md]: Ooh i forgot! Added

File: source/2019-12-13-the-ember-times-issue-128.md [hunk 5]
```diff
@@ -24,6 +25,14 @@ READMORE
...
+
+Unfortunately there was not a well defined standard for making accessible drag and drop controls from organizations such as the [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/). He turned to [Andrew A Lee (@drewlee)](https://github.com/drewlee) who created a comprehensive list on what accessible features should be expected on a drag and drop utility and even a [CodePen demo](https://codepen.io/drewlee/project/full/XWNLeE). These accessible controls have now landed as part of the [2.0.0 release](https://github.com/adopted-ember-addons/ember-sortable/releases/tag/v2.0.0) of the [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) addon. Check out the accessible controls at the [demo page](https://adopted-ember-addons.github.io/ember-sortable/demo/) today!
```
Comment (MelSumner) [inline source/2019-12-13-the-ember-times-issue-128.md]:
```suggestion
Unfortunately there was not a well-defined standard for making accessible drag and drop controls from organizations such as the [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/). He turned to [Andrew A Lee (@drewlee)](https://github.com/drewlee) who created a comprehensive list on what accessible features should be expected on a drag and drop utility and even a [CodePen demo](https://codepen.io/drewlee/project/full/XWNLeE). These accessible controls have now landed as part of the [2.0.0 release](https://github.com/adopted-ember-addons/ember-sortable/releases/tag/v2.0.0) of the [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable) addon. Check out the accessible controls at the [demo page](https://adopted-ember-addons.github.io/ember-sortable/demo/) today!
```

**Pull request merged**